doc/development_process/proposals.rst

```diff
@@ -0,0 +1,26 @@
+
+Proposals and RFCs
```

Comment:
Side comment: (I know this text is mostly a copy but) I think it would be good to eventually add a paragraph in line with what you mentioned in last TSC: that any mayor change or mayor addition should be brought up to the TSC. So the TSC is informed, can provide guidance, and the change, if accepted, is accounted for in the roadmap.

Reply:
yes, will add.